src/pages/billing/List.js

```diff
@@ -145,160 +179,112 @@ const NewBillingProjectModal = ajaxCaller(class NewBillingProjectModal extends C
           ])
         ])
       ]),
-      submitting && spinnerOverlay
+      isBusy && spinnerOverlay
     ])
   }
 
   async submit() {
     const { onSuccess, ajax: { Billing } } = this.props
     const { billingProjectName, chosenBillingAccount, existing } = this.state
 
+    Utils.withBusyState(v => this.setState({ isBusy: v }))
```

Comment:
@panentheos is this the correct way to use `withBusyState` when it is not a part of a `_.flow`?

Reply:
ah, I misunderstood. You want to feed this the try block below as an async function as its second argument